docs/framework/wcf/feature-details/how-to-make-x-509-certificates-accessible-to-wcf.md

```diff
@@ -53,8 +53,6 @@ To make an X.509 certificate accessible to Windows Communication Foundation (WCF
         |--------------|----------------------|  
         |Client (console or WinForms application).|Currently logged in user.|  
         |Service that is self-hosted.|Currently logged in user.|  
-        |Service that is hosted in IIS 6.0 ([!INCLUDE[ws2003](../../../../includes/ws2003-md.md)]) or IIS 7.0 (Windows Vista).|NETWORK SERVICE|  
```

Comment:
I'd keep this one until we revisit this article to see what is the identity for IIS 7+